templates/helpers.js

```diff
@@ -402,14 +402,18 @@ DocMapInfo.prototype.getCurrentTree = function(){
     //
     var getChildren = this.getChildren.bind(this),
         getNestedDocObject = this.getNestedDocObject.bind(this);
-
+    
+    var self = this;
     var cur = this.getCurrent();
-
     var curChildren = this.getNestedChildren(cur);
-
     this.getParents(cur, function(docObject){
         curChildren = getChildren(docObject).map(function(docObject){
             if(docObject.name === cur.name) {
+                if(cur.subchildren){
+                    curChildren.forEach(function(child, i){
+                        if(child.docObject) curChildren[i].children = self.getNestedChildren(child.docObject);
```

Comment:
You're right. For some reason, I thought `forEach` made a copy of each object but that would be rediculous. Updated the PR.